test/units/modules/network/ovs/test_openvswitch_bridge.py

```diff
@@ -155,6 +160,21 @@ def test_openvswitch_bridge_present_creates_fake_bridge(self):
         self.execute_module(changed=True, commands=commands,
                             test_name='test_openvswitch_bridge_present_creates_fake_bridge')
 
+    @patch.object(openvswitch_bridge, 'map_config_to_obj')
```

Comment:
With pytest, we usually go for fixtures so that you'd have a separate fixture function patching staff and yielding some object for testing and the test function will be more readable and contain actual testing code with assertions.
For example:
```python
from units.compat.mock import MagicMock


@pytest.fixture
def patched_openvswitch_bridge(monkeypatch):
    with monkeypatch.context() as mp_ctx:
        mp_ctx.setattr(openvswitch_bridge, 'map_config_to_obj', MagicMock())
        # code before "yield" is executed before test function
        yield openvswitch_bridge  # "yield"ed value is what test function receives as an input
        # code after "yield" is executed after test function
        # and can be used for cleanup, we don't need this because
        # it'll be taken care of by the context manager


def test_openvswitch_bridge_updates_vlan(self, patched_openvswitch_bridge):
    ...  # patched_openvswitch_bridge is patched inside this test method
```